### DISCUSSION TITLE
Printing thin object seems bugged

### DISCUSSION THREAD

**Discussion by d-schmidt** [smartavionics/Cura#107] [Q&A] [answered]
I don't know yet if this a problem with your build or a general cura problem.
It slices this really bad.
The walls in the object are 0.5mm. Had to enable "print thin walls" to get anything.

Any idea whats going on here? @smartavionics

Line Width 0.4 results in double printing of the socket:
![grafik](https://user-images.githubusercontent.com/3226483/117493862-d71c7b80-af73-11eb-9575-a118324dd1f2.png)
and 0.33mm walls higher up:
![grafik](https://user-images.githubusercontent.com/3226483/117493928-ed2a3c00-af73-11eb-842e-69f7f2ac9ded.png)

Setting line width to .5 causes artifacts and holes:
![grafik](https://user-images.githubusercontent.com/3226483/117494060-19de5380-af74-11eb-8f5c-b271d48e1a5c.png)

Model:
[quartz2_3mmLED.zip](https://github.com/smartavionics/Cura/files/6443644/quartz2_3mmLED.zip)
![grafik](https://user-images.githubusercontent.com/3226483/117493760-b5bb8f80-af73-11eb-915a-bf6aa3ee685e.png)

**Comment by smartavionics** [comment, marked as answer]
Yes, that's standard behaviour for Cura, it has never been able to handle thin walls well. However, the UM folks are working on a new implementation of wall printing that is intended to fix this issue. See the Arachne discussions at https://github.com/Ultimaker/Cura/discussions for more information.
